src/status_im/subs.cljs

```diff
@@ -347,6 +347,21 @@
           (sort-by #(get names (get % 0)))
           (sort-by #(visibility-status-utils/visibility-status-order (get % 0)))))))
 
+(re-frame/reg-sub
+ :communities/featured-communities
+ :<- [:communities/enabled?]
+ :<- [:search/home-filter]
+ :<- [:communities]
+ (fn [[communities-enabled? search-filter communities]]
+   (filterv
+    (fn [{:keys [name featured id]}]
+      (and (or featured (= name "Status")) ;; TO DO: remove once featured communities exist
```

Comment:
I believe there's a typo in this TODO "type", the convention is to not add spaces, just `TODO: ...`. I'm just mentioning this because it's easier to subsequently search for this string in the condebase if it's always the same.